ci-operator/step-registry/openshift/upgrade/aws/sdn/openshift-upgrade-aws-sdn-workflow.yaml

```diff
@@ -0,0 +1,25 @@
+workflow:
+  as: openshift-upgrade-aws-sdn
+  steps:
+    allow_best_effort_post_steps: true
+    pre:
+    - ref: ipi-install-hosted-loki
+    - chain: ipi-conf-aws
+    - ref: sdn-conf
+    - chain: ipi-install-stableinitial
+    test:
+    - ref: openshift-e2e-test
+    post:
+    - ref: openshift-e2e-test-capabilities-check
+    - chain: ipi-aws-post
```

Comment:
In `openshift-upgrade-aws-ovn-workflow.yaml` on [master](https://github.com/openshift/release/blob/e34ef147ad81e3bce9516abe93b956185d72426e/ci-operator/step-registry/openshift/upgrade/aws/ovn/openshift-upgrade-aws-ovn-workflow.yaml), I see:

```
    - chain: gather-network
    - chain: gather-core-dump
    - chain: ipi-aws-post
```

do we want those for the sdn workflow too?

Reply:
yeah, probably would make sense to add these, but let's do this as a follow up PR. I'm currently in the middle of trying to get all these jobs renamed to use -sdn- and don't want to add a new variable to my chaos. I'll make a jira for myself to keep track of this:

https://issues.redhat.com/browse/SDN-3213